packages/btc/src/transaction/utxo.ts

```diff
@@ -56,3 +59,62 @@ export function utxoToInput(utxo: Utxo): TxInput {
 
   throw new TxBuildError(ErrorCodes.UNSUPPORTED_ADDRESS_TYPE);
 }
+
+/**
+ * Fill pubkey for P2TR UTXO, and optionally throw an error if pubkey is missing
+ */
+export function fillUtxoPubkey(
+  utxo: Utxo,
+  pubkeyMap: Record<string, string>, // Record<address, pubkey>
```

Comment:
You can define a new type `type pubkey = string` which can make the code more clear and the comment can be removed

Reply:
It would be hard to mention the `Address` type everywhere, or do you mean we should only define a type proxy for `type Pubkey = string`?

Reply:
`type pubkey = string`  can be used to replace comment and make the code more clear and simple.

Reply:
The problem here is if `Address` and `Pubkey` exist, should the type definition of `from`, `changeAddress` and `fromPubkey` also be refactored with the type proxies? If yes, there could be a lot of work, for example: 
```ts
- from: string;
+ from: Address;
```

Reply:
I think defining some common types to indicate the data types is helpful and clear like https://github.com/ckb-cell/rgbpp-sdk/blob/295df7248ce7c82301990d871863850429835007/packages/ckb/src/types/common.ts.

However, it depends on you and I just give some suggestions.

Reply:
I think improving the code readability is nice but the change is relatively huge, so I've opened a discussion for this: https://github.com/ckb-cell/rgbpp-sdk/issues/235, feel free to leave your comments @Flouse @ahonn @Dawn-githup